dev-notes/minimal-os-for-fobnail.md

```diff
@@ -0,0 +1,203 @@
+# Minimal OS for Fobnail project
+
+The Fobnail project aims to provide a reference architecture for building
+offline integrity measurement servers on the USB device and clients running in
+Dynamically Launched Measured Environments (DLME). It should allow the Fobnail
+owner to verify the trustworthiness of the running system before performing any
+sensitive operation.
```

Comment:
What do you think about description of the project, which I proposed [here](https://github.com/fobnail/docs/pull/29/files#diff-b4d68dc855d0f9476d3f2ee343853bd21bf82ea9960d0cf06661baa244439dd6R3) ?

Also do we really have to repeat that intro, maybe we should just refer to index or detailed description?